source/logrepl/cdc.go

```diff
@@ -154,21 +154,26 @@ func (i *CDCIterator) attachSubscription(ctx context.Context, conn *pgx.Conn) er
 		}
 	}
 
-	keyColumn, err := i.getKeyColumn(ctx, conn)
-	if err != nil {
-		return fmt.Errorf("failed to find key for table %s (try specifying it manually): %w", i.config.TableName, err)
+	var err error
+	keyColumnMp := make(map[string]string, len(i.config.Tables))
+	for _, tableName := range i.config.Tables {
+		// Call function and store the result in the map
+		keyColumnMp[tableName], err = i.getKeyColumn(ctx, conn, tableName)
+		if err != nil {
+			return fmt.Errorf("failed to find key for table %s (try specifying it manually): %w", tableName, err)
```

Comment:
IIUC we plan to remove the key column, so the error message is a bit misleading. Although, thinking about it, we could accept comma separated pairs, where each pair is separated by a colon, e.g.:
```yaml
table: "employees,users,addresses",
key: "employees:emp_id,users:id",
```
And only if we don't find a manually specified key column, we fall back to fetching it from the DB (like for `addresses` in the example above).